master/pom.xml

```diff
@@ -584,7 +584,7 @@
       <dependency>
         <groupId>dom4j</groupId>
         <artifactId>dom4j</artifactId>
-        <version>1.6.1</version>
+        <version>1.6.1-jboss</version>
```

Comment:
Can this be bumped to 2.1.0? https://dom4j.github.io/

Reply:
groupId have its diferent.
Would try with org.dom4j

Reply:
Fixed at #1290855